File: boj_old/1_io/1924_1.cpp
#include <cstdio>

const char *days[] = { "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT" };
const int dates[] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };

int main()
{
    int m, d, diff = 0;
    scanf("%d %d", &m, &d);

    for (int i = 1; i < m; i++)
        diff += dates[i - 1];
    diff += d;
    puts(days[diff % 7]);
}

File: js/1-io/_1924-1.js
const _input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");
let _input_idx = 0;
const input = () => _input[_input_idx++];

const DAYS = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];

let [x, y] = input().split(" ").map(e => +e);
console.log(DAYS[new Date("2007", x - 1, y).getDay()]);
